certbot-dns-alwaysdata/docs/api/dns_alwaysdata.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`certbot_dns_alwaysdata.dns_alwaysdata`
+------------------------------------
```

Comment:
Could you try generating the documentation (e.g., using `make html` from the `certbot-dns-alwaysdata/docs` directory with an active `venv`) and fixing any warnings/errors? I suspect you'll get one for this line of hyphens being too short. (ReStructured Text can be a bit picky about formatting.)